bindings/node/src/index.ts

```diff
@@ -0,0 +1,76 @@
+import * as bindings from 'bindings';
+
+const mc = bindings('mongocrypt');
+
+interface MongoCryptKMSRequest {
+  addResponse(response: Buffer): void;
+  status: MongoCryptStatus;
+  bytesNeeded: number;
+  kmsProvider: string;
+  endpoint: string;
+  message: Buffer;
```

Comment:
```suggestion
  readonly status: MongoCryptStatus;
  readonly bytesNeeded: number;
  readonly kmsProvider: string;
  readonly endpoint: string;
  readonly message: Buffer;
```

(All of these only have getters? ditto below)